### DISCUSSION TITLE
[Feature] Make shared albums awesome when accessed on mobile web

### DISCUSSION THREAD

**Discussion by klaus1k** [immich-app/immich#11731] [Feature Request]
### The feature

I often take vacations with friends and want to share my photos with them. As this would be the only times they use immich atm, they do not want to install the app. So they open the link to the shared album on their smartphone/mobile. Right now there is room for improvement compared to desktop web and mobile app, which already have very nice UX.

Following improvements could be done:
- [x] Images should bei smaller in grid overview, like in the mobile app
 #16902
- [x] The margin on the left and right of the grid should be removed, like in the mobile app
- [x] "Scrobble" timeline in the right border should not be displayed on small displays/mobile - there are lots of wrong clicks with accidental touches on the timeline (it is perfect like it is implemented on the mobile app)
#16856 
- [x] when you tap on a photo in album view on iphone it does not open in full screen, it gets "checkmarked" instead - it needs a second tap to open in fullscreen; on android the photo gets "checkmarked", but opens in full screen anyway
#16906
- [x] The immich logo in the header is still a bit small and the "immich" text is missing
- [x] swipe navigation between full screen photos would be the last piece of the puzzle for an awesome experience! #11888 
- [ ] Added (March 2025): When a photo is viewed in fullscreen, double-tap zooms in all the way. It would be nice if it could replicate the behaviour of the app, where it zooms in about 30% each double tap.

Screenshot of the immich demo (shared album link opened on Pixel 8 Pro in Vanadium/Chrome):
<img src="https://github.com/user-attachments/assets/56d1363b-4042-451d-bcec-887f7874a858" width="300" />
Similar issues/FR/PR I found:
- (old) issue #7941
- Closed PR #4182

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by klaus1k** [comment]
updated, now that swipe navigation is merged (yay!)

**Comment by jzavalaneri** [comment]
This would be great. 

I've been using Immich now for several months with the wife and kids. They are now starting to create albums and share them out with friends and family that don't have the app. 

This has been a big ask from us and a big complain from those that use the web on their phone to access shared albums.

**Comment by histefanhere** [comment]
Relevant earlier discussion post: #4798

**Comment by midzelis** [comment]
Hey - thanks for this list. I'm also interested in improve the mobile web experience - I'll link my PRs as I make them

**Reply by klaus1k** [reply]
Awesome, thanks a lot for your work!
Really looking forward to an optimized web-view on mobile. I'll add your PRs in the original post.

**Comment by midzelis** [comment]
This PR has gone stale, but there might still be some things can be applied: #8776

**Comment by klaus1k** [comment]
I just added the following to the list in the original post:

Added (March 2025): When a photo is viewed in fullscreen, double-tap zooms in all the way. It would be nice if it could replicate the behaviour of the app, where it zooms in about 30% each double tap.

**Comment by klaus1k** [comment]
Thanks everyone, especially @midzelis! mobile web experience is much better now!
